driver-scala/src/main/scala/org/mongodb/scala/MongoCredential.scala

```diff
@@ -204,7 +204,7 @@ object MongoCredential {
    *
    * @param userName the non-null user name
    * @return the credential
-   * @see [[http://docs.mongodb.org/manual/core/authentication/#kerberos-authentication GSSAPI]]
+   * @see [[https://www.mongodb.com/docs/manual/core/authentication/#kerberos-authentication GSSAPI]]
```

Comment:
Looks like this refers to an old target.
Instead, maybe the link should be:
https://www.mongodb.com/docs/manual/core/kerberos/
or if you want to link directly to GSSAPI through AD auth:
https://www.mongodb.com/docs/manual/tutorial/kerberos-auth-activedirectory-authz/